tensorboard/backend/event_processing/db_import_multiplexer.py

```diff
@@ -108,10 +108,11 @@ def AddRunsFromDirectory(self, path, name=None):
       tf.logging.info('Processing directory %s', subdir)
       if subdir not in self._run_loaders:
         tf.logging.info('Creating DB loader for directory %s', subdir)
+        exp_name, run_name = self._get_exp_and_run_names(path, subdir, name)
```

Comment:
Mind renaming `exp_name` to `experiment_name` in all uses, for clarity?
(cf. our Python style guide: “eschew abbreviation […] do not use
abbreviations that are ambiguous or unfamiliar to readers outside your
project, and do not abbreviate by deleting letters within a word”).